app/repositories/foundation-info/foundationInfo.repository.ts

```diff
@@ -0,0 +1,70 @@
+import type { Locale } from 'next-intl';
+
+import type {
+  ContactInfoData,
+  FoundationNameData,
+  PublicInfoData,
+  SupportButtonLinkData
+} from '~/types/types/foundationInfo.type';
+
+import { BrandingInfo } from '~/models/foundation-info/foundationInfoBranding';
+import { ContactInfo } from '~/models/foundation-info/foundationInfoContact';
+import { PublicInfo } from '~/models/foundation-info/foundationInfoPublic';
+import { hrefSchema } from '~/validators/constants';
+import { brandingInfoSchema, contactInfoSchema, publicInfoSchema } from '~/validators/foundationInfo.schema';
+
+export const contactRepository = {
+  async getContactInfo(): Promise<ContactInfoData> {
+    const result = await ContactInfo.findOne({ slug: 'contact-info' })
+      .select('email phone contactButtonLink socialLinks')
```

Comment:
```suggestion
      .select('email phone contactButtonLabel socialLinks')
```
Maybe this correct ? 
Because in `faundationInfoContact` model i saw this field instead of `contactButtonLink `

Or is here correct variant so change it in `faundationInfoContact` to correct one
Because I think that the link should be returned here and this comes

<img width="615" alt="Знімок екрана 2025-06-09 о 18 55 16" src="https://github.com/user-attachments/assets/1f821998-ae7a-44ef-92ae-e71c6967b1f7" />

Reply:
I’ll remove the contactButtonLink — I just re-read the user story, and the button is supposed to open a modal window, not navigate anywhere.

Reply:
OK, i see that you delete it now from `faundationContactInfo` but here you forgot to delete it, so please delete it here too